docs/configuration/index-config.md

```diff
@@ -80,10 +80,9 @@ The doc mapping defines how a document and the fields it contains are stored and
 | `field_mappings` | Collection of field mapping, each having its own data type (text, binary, datetime, bool, i64, u64, f64).   | [] |
 | `mode`        | Defines how quickwit should handle document fields that are not present in the `field_mappings`. In particular, the "dynamic" mode makes it possible to use quickwit in a schemaless manner. (See [mode](#mode)) | `lenient`
 | `dynamic_mapping` | This parameter is only allowed when `mode` is set to `dynamic`. It then defines whether dynamically mapped fields should be indexed, stored, etc.  | (See [mode](#mode))
-| `tag_fields` | Collection of fields already defined in `field_mappings` whose values will be stored in a dedicated `tags` (1) | [] |
+| `tag_fields` | Collection of fields already defined in `field_mappings` whose values will be stored in a dedicated `tags`. [Learn more about tags](../concepts/querying.md#tag-pruning). | [] |
```

Comment:
Nitpick: We can probably improve the wording around the tag_fields sections so it's a bit clearer up from reading.

```suggestion
| `tag_fields` | Collection of fields already defined in `field_mappings` whose values will be stored as part of the `tags` metadata. [Learn more about tags](../concepts/querying.md#tag-pruning). | [] |
```

Reply:
yes that's better